pdns/common_startup.cc

```diff
@@ -183,7 +187,8 @@ void declareArguments()
   ::arg().set("domain-metadata-cache-ttl","Seconds to cache domain metadata from the database")="60";
 
   ::arg().set("trusted-notification-proxy", "IP address of incoming notification proxy")="";
-  ::arg().set("slave-renotify", "If we should send out notifications for slaved updates")="no";
+  ::arg().set("slave-renotify", "If we should send out notifications for secondaried updates")="no";
+  ::arg().set("secondary-renotify", "If we should send out notifications for secondaried updates")="no";
```

Comment:
Not a big fan of secondaried

Reply:
Hmm, do you have an alternative suggestion?

Reply:
Can you explain what it is?

Reply:
secondaried means 'The secondary has transferred, from the primary, a new version of the zone.'

Reply:
Does this fit your intent?
```suggestion
  ::arg().set("secondary-renotify", "If we should send out notifications when secondary receives updates")="no";
```

Reply:
it's close, something with 'after' would be even closer

Reply:
Hmm, can you explain a bit?

Note:
`when` to me implies that it's the secondary sending the update

```suggestion
  ::arg().set("secondary-renotify", "If we should send out notifications after secondary receives updates")="no";
```

I worry that `after` doesn't indicate who is doing the sending

(I'm not particularly confident on this.)

Reply:
'Whether we should send out notifications after receiving updates from the primary'

Reply:
from IRC, proposals to also rename the option while we're at it, because the current name is not 100% accurate: `notify-for-secondary`, `notify-by-secondary`, `notify-as-secondary`

Reply:
current proposal: secondary-do-notify: Whether this secondary sends out notifications after receiving updates from the primary for a zone

Reply:
we should say 'zone transfers' or 'XFR(s)' instead of 'updates'

Reply:
And `a primary` instead of `the primary` :-)

Reply:
s/the primary/a primary/